nbconvert/templates/latex/base.tplx

```diff
@@ -13,9 +13,7 @@ This template does not define a docclass, the inheriting class must define this.
     ((* block docclass *))((* endblock docclass *))
     
     ((* block packages *))
-    \usepackage[T1]{fontenc}
-    % Nicer default font (+ math font) than Computer Modern for most use cases
-    \usepackage{mathpazo}
+    \usepackage{fontspec}
```

Comment:
Some nbconvert users still choose to use pdflatex, and fontenc does work on XeLaTeX (with warnings), but fontspec does not work at all on pdflatex.

Solution:
```latex
\usepackage{iftex}
\ifPDFTeX {
    \usepackage[T1]{fontenc}
    \usepackage{mathpazo}
} \else {
    \usepackage{fontspec}
} \fi
```
This will continue to use fontspec even on lualatex too.

Reply:
This seems like a very good idea! A similar if structure could then also be used for `\usepackage[utf8]{inputenc}` to ensure compatibility across latex compilers.

Reply:
That won't be necessary for inputenc. Inputenc is ignored in XeLaTeX and it defaults to utf8 in recent PDFLaTeX. See https://www.latex-project.org/news/latex2e-news/ltnews28.pdf and https://tex.stackexchange.com/questions/13067/utf8x-vs-utf8-inputenc.

> This means that with new documents one can assume UTF-8 input and it is no longer required to always specify \usepackage[utf8]{inputenc}. But if this line is present it will not hurt either.

Reply:
Ah, I did not know that yet. That makes it a lot simpler.